app/upgrades.go

```diff
@@ -51,12 +50,6 @@ func (app *UmeeApp) registerUpgrade4_2(upgradeInfo upgradetypes.Plan) {
 	})
 }
 
-// performs upgrade from v4.1.0-rc2 to v4.1.0-rc3
-func (app *UmeeApp) registerUpgrade4_1_0rc3(_ upgradetypes.Plan) {
-	const planName = "v4.1.0-rc3"
-	app.UpgradeKeeper.SetUpgradeHandler(planName, onlyModuleMigrations(app, planName))
-}
```

Comment:
@toteki , this is not relevant to the main branch, should be only in the release branch, as the main branch is already ahead.